input/intro-notes/StructureDefinition-au-core-practitioner-notes.md

```diff
@@ -32,27 +32,27 @@
 
 The following search parameters **SHALL** be supported:
 
-1. **SHALL** support searching a practitioner by an identifier using the **[`identifier`](https://hl7.org/fhir/R4/practitioner.html#search)** search parameter:
+1. **SHALL** support searching using the **[`identifier`](https://hl7.org/fhir/R4/practitioner.html#search)** search parameter:
 
     `GET [base]/Practitioner?identifier={system|}[code]`
 
     Example:
     
       1. GET [base]/Practitioner?identifier=http://ns.electronichealth.net.au/id/hi/hpii/1.0\|8003619900015717
+      1. GET [base]/Practitioner?identifier=8003619900015717
 
-    *Implementation Notes:* Fetches a bundle containing any Practitioner resources matching the identifier ([how to search by token](http://hl7.org/fhir/R4/search.html#token))
+    *Implementation Notes:* Fetches a bundle containing any Practitioner resources matching the identifier ([how to search by id of the resource](https://hl7.org/fhir/r4/search.html#id))
```

Comment:
I think this should still be how to search by token.

Reply:
fixed.